src/app/pages/Menu/Menu.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import MenuLink from '../../components/MenuLink/MenuLink';
+import style from './Menu.module.css';
+
+export default function Menu(): JSX.Element {
```

Comment:
I would rename this to `Landing` because it feels more like a landing page, also adjust the route